### PR TITLE
Update taxable.rb

### PR DESCRIPTION
**Pull request by swatosh**
make nothing in the "cart" state taxable

### CODE REVIEW

File: app/models/concerns/spree_taxjar/taxable.rb [hunk 1]
```diff
@@ -5,6 +5,8 @@ module Taxable
     private
 
     def taxjar_applicable?(order)
+      return false if order.state = 'cart'
```
Comment (iandusenbury) [inline app/models/concerns/spree_taxjar/taxable.rb]: Is this setting the state to 'cart'? Don't you need `==`?

Comment (swatosh) [reply app/models/concerns/spree_taxjar/taxable.rb]: Yes, 'cause I typoed it. Standby

Comment (swatosh) [reply app/models/concerns/spree_taxjar/taxable.rb]: doh. try again

**Comment by swatosh** [comment]
I prolly deserve that for writing it in the browser instead of checking it out

**Review by iandusenbury — APPROVED** [review]

**Pull request merged**